src/code/fv-components/location-indicator.js

```diff
@@ -0,0 +1,18 @@
+import React, {PropTypes} from 'react';
```

Comment:
There's certainly a gray area and different engineers will decide differently in different circumstances. One of the criteria I use when thinking about it is how well does the component stand on its own -- for instance, how likely is it to be useful in another context? In the case of the stable drakes, the effect was intended to be specific to the stable, so there's an argument for having the stable define the effect. In this case, I feel like it's the HUD's job to lay out the components on the HUD, but the components themselves should be self-contained.